css/style.css

```diff
@@ -0,0 +1,97 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body,
+html {
+  height: 100%;
+  width: 100%;
+  background-color: #F1F5F9;
+  color: #1E293B;
+  font-family: 'Rubik',
+  sans-serif;
+}
+
+main {
+  margin: 140px auto;
+  width: 100%;
+  max-width: 650px;
+  height: 100vh;
+}
+/* Random Quote Section*/
+.todays-motivation{
+  font-size:14px;
+}
+.random-quote{
+  background-color: #fff;
+  margin: 20px 0px 80px 0px;
+  height: 80px;
+  align-items: center;
+  padding: 25px 20px;
```

Comment:
Look at all PX margin/padding numbers. Make sure each number is divisible by 4s.
`4, 8, 12, 16, 20, 24, 32, 64` etc

Reply:
I didn't tell you this before you started, but it's good to stick with a scale when using any spacing/font-size increments.

Use the same general scale for font-sizes.